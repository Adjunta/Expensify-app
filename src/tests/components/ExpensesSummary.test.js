import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import { ExpensesSummary } from '../../components/ExpensesSummary';

test('should display count and total of 1 expense', () => {
  const wrapper = shallow(<ExpensesSummary expensesCount={1} expensesTotal={235}/>);
  expect(toJSON(wrapper)).toMatchSnapshot();
})

test('should display count and total of 2 expenses', () => {
  const wrapper = shallow(<ExpensesSummary expensesCount={23} expensesTotal={6598712168}/>);
  expect(toJSON(wrapper)).toMatchSnapshot();
})