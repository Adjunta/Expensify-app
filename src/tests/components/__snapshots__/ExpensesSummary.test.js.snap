// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should display count and total of 1 expense 1`] = `
<div>
  <h1>
    Viewing 
    1
     
    expense
     totalling 
    $2.35
     
  </h1>
</div>
`;

exports[`should display count and total of 2 expenses 1`] = `
<div>
  <h1>
    Viewing 
    23
     
    expenses
     totalling 
    $65,987,121.68
     
  </h1>
</div>
`;
